tests/static_tests/functional/static_if_tests.cpp

```diff
@@ -64,4 +66,18 @@ TEST_CASE("static_if") {
             })();
         REQUIRE(called == -1);
     }
+    {  //  simple true
+        auto called = false;
+        call_if_constexpr<std::true_type::value>([&called] {
```

Comment:
if we have a bug when lambda is called twice this bug will never be detected with this UT. Better change `bool called` to `int called`, make `++called` inside lambda and `REQUIRED(called == 1)`

Reply:
I can change this one. Though it's not different from the other `static_if` tests above.